### PR TITLE
Feature/polls endpoints

### PR DESCRIPTION
**Pull request by picklechips**
Adds relevant endpoints to:

- Add poll to a post
- Vote on a poll
- Add an option to a poll
- Get poll from post

### CODE REVIEW

File: controllers/postController.js [hunk 1]
```diff
@@ -202,4 +203,126 @@ router.get('/:id/image', verifyToken, (req, res) => {
...
+        .catch((err) => {
+          res.status(500).json(err.toString());
+        });
+    })
+    .catch((err) => {
+      res.status(500).json(err.toString());
+    });
+});
+
+router.post('/:id/poll/vote', verifyToken, (req, res) => {
```
Comment (scholvat) [inline controllers/postController.js:296]: Is there any way to unvote for an option?

Comment (picklechips) [reply controllers/postController.js:296]: Lol somehow I did not think about that yeah I'll have to add that

Comment (picklechips) [reply controllers/postController.js:296]: Updated

**Comment by picklechips** [comment]
Moving PR to cguc

**Pull request closed without merging**